go/capship/commands/server.go

```diff
@@ -0,0 +1,139 @@
+/*
+Copyright 2018 The cap Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package commands
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/IBM/cap/go/log"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	alertsDirectory = "alerts/"
+	feedsDirectory  = "feeds/"
+)
+
+// Server - capship host
+type Server struct {
+	config  *Config
+	context context.Context
+}
+
+// New - Allocates a new instance of a capship server
+func New(ctx context.Context, config *Config) (*Server, error) {
+	switch {
```

Comment:
why a single case switch instead of an if?

Reply:
was going to "add" more config error checks here"